pageserver/src/layered_repository.rs

```diff
@@ -606,88 +608,70 @@ impl LayeredRepository {
         let now = Instant::now();
 
         // grab mutex to prevent new timelines from being created here.
-        let _gc_cs = self.gc_cs.lock().unwrap();
+        let _gc_cs = self.gc_cs.lock().unwrap(); // BLOCKED HERE
```

Comment:
Some debugging leftovers? Worth a bit bigger elaboration, if not.

Reply:
Just noticed another one below, so feels like a leftover after all.

Reply:
Cleaned this up in all places